md2man/roff.go

```diff
@@ -1,252 +1,281 @@
 package md2man
 
 import (
-	"bytes"
 	"fmt"
-	"html"
+	"io"
+	"os"
 	"strings"
 
 	"github.com/russross/blackfriday"
 )
 
-type roffRenderer struct {
+// RoffRenderer implements the blackfriday.Renderer interface for creating
+// roff format (manpages) from markdown text
+type RoffRenderer struct {
```

Comment:
Is there a good reason for this change? This breaks the package API.

Reply:
Ha. I was bullied into it by the linter. Of course, the linter can be overcome. I wasn't thinking much about API, but you're correct.